pkgs/development/libraries/utf8proc/default.nix

```diff
@@ -15,6 +15,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [
     "-DBUILD_SHARED_LIBS=ON"
+    "-DUTF8PROC_ENABLE_TESTING=ON"
```

Comment:
doCheck  = true is the default so no need to add it probably but DUTF8PROC_ENABLE_TESTING could be enabled depending on doCheck

Reply:
`doCheck = true` is [not (yet) the default](https://github.com/NixOS/nixpkgs/blob/12f367b51cafa511161b1af76f9ccc47c2dde6cb/pkgs/stdenv/generic/make-derivation.nix#L63)

Reply:
my bad, I must have confounded with python